<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/bin/conda-lint", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/run.py", line 107, in main
    result = linter.lint(
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/lint/__init__.py", line 620, in lint
    msgs = self.lint_file(
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/lint/__init__.py", line 692, in lint_file
    messages = self.lint_recipe(
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/lint/__init__.py", line 749, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/lint/__init__.py", line 247, in run
    self.check_recipe(recipe)
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/lint/check_build_help.py", line 554, in check_recipe
    o = int(path.split("/")[1])
ValueError: invalid literal for int() with base 10: 'host'

ERROR conda.cli.main_run:execute(47): `conda run conda-lint /tmp/abs_66emlkk3yk/clone` failed. (See above for error)
</details>

--- a/abs.yaml
+++ b/abs.yaml
@@ -1,0 +1,4 @@
+# the conda-build parameters to use for disabling --skip-existing
+build_parameters:
+  - "--suppress-variables"
+

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -34,8 +34,6 @@ outputs:
   - name: libcryptominisat
     build:
       skip: True  # [win and vc<14]
-      ignore_run_exports:
-        - zlib
       run_exports:
         - {{ pin_subpackage("libcryptominisat", max_pin="x.x") }}
     script: install-lib.sh  # [unix]
@@ -67,8 +65,6 @@ outputs:
     build:
       number: {{ build }}
       skip: True  # [win and vc<14]
-      ignore_run_exports:
-        - zlib
     script: install-py.sh   # [unix]
     script: install-py.bat  # [win]
     requirements:

